sdk/security/keyvault/azcertificates/options.go

```diff
@@ -83,7 +78,6 @@ type ListCertificatePropertiesOptions struct {
 // ListCertificatePropertiesVersionsOptions contains the optional parameters for the Client.NewListCertificatePropertiesVersionsPager
 // method.
 type ListCertificatePropertiesVersionsOptions struct {
-	// placeholder for future optional parameters
```

Comment:
Did the generator do this? It's weird that other types still have this comment

Reply:
fixed in 26fa941

Reply:
The comment isn't so important that we must ensure it's present, it's just the apparent inconsistency in the generator that's interesting

Reply:
No, it's not an issue in the generator. The MaxResults parameter was there but was removed in a transform. Modified the transform to add the comment.